web/statik/statik.go

```diff
@@ -488,6 +488,363 @@ jy/+fwAAAP//TfvNpxmuAQA=
+xosNsK35LtWCEq/FEGw6dj+t2WKYy822OKVzppVCg7HFu1yJ+4Gz3PvnyfSW+SnS
+BAtR3jYorYR8eZg9ZoXBNQ3ffIlCavBcZO9fytKrV7378QW9+QJZjWNxnA3cRG5N
+v3hz8Vxfny/fO/SOP9W9wjAMRYtYJvzRcBM3cLr0F9Iqj/VcRAkGx1Z5aLPnBc22
+WZ+4FXyuP87j+eObc00K7XhcHfFRCNCDhpMMEvo1DpVfGypYt182QLqjE8Ni+2a4
+t7lzpRt5PL/2bbsOvVolW6nhP/wxPN/YJLwZFO/69CXa+a7SBxrMInXbOAptUahF
+KZbaNCTPwIZ0CJ0/z/HeV+KBODGipMECOK3JASgfZ1H0Hm3cqNrNteDbgdWADlOY
+sGHGl0kxFHyfDvoRRpYbGD4PgR5GUo0L1pvEcfmXeuOfEy7us14XJZCzjZsYvmuO
+G4hwh8omoZ9IK/BvXgY9NgJZ/YnihvRVJJrimBwNdPVHORCMcOTMcT+qFkRrDPAj
+xcRco3BbKySERL9mWWCbF00SPbx/+phcInUQ+gWxn9pxRKoxb0Rx+wrkIiWE3JQQ
+0rj8W2XApWhUh9isP78i50tFWHIm3/cPrdIkF/cDh71dr1rpxgi5STzjCqkyvBKK
+adpw6uk3X4rxbDg3cg0J2FKsGMH8vjgCzmU9pgONNnm7C3bRuzGh0NZuiHZ2ApYX
+r8bnMjvkGkVd4hr8OcKMBYs4NSQXzRobP3RChliteAeJCtz+a8rd7XXhG0PA4d29
+Q4USIPPWhuX8bsF9voaO3d6DlWcBcuuUoCNCRA816szDBWVEz/ZzQNRqxm9t/3BG
+2RKaopQvtS1ebMsM0ldSlIadvoHUCDxFAcDoltZClf9aM5pGPX3c2PQNd//cNYYp
+yH6v+86Gmk8mklxM6PjEVMb0bJ/fMkiFH9Z6xm1pyy6zi4edyAiCbHI4+PYr+oXd
+MwxLInjvNL1062BH9t6ODN96xayYRGHgUO5tVfF+MrTNUIVlQ1nAjs0Hev0XWpOA
+Ks0ZXX6RbEfaRuXGC9hFYjtIWGy6dmDaXQ6XrL4SXalmgNDcyPDo0I91BkgdewpV
+tUNk37thGlf7qpoDsBsm1o867FYKIOvICKyWkXBPWYIrs8cP5fG8G0DVm4q/tXH9
+xI5Ga8TegR3HL4VXHC6Hjm2RS/v5Ei0acH/naKTaiyvLB7sdTuUi5F5RFIi3jIfE
+WRs1My8Gg2nHK0O/fCOxVy/5ISRUd1bVj8A+JQwPsnHL12tMVUEj1u5YLZIuq4VF
++JnG43LjurwoMB4Gby2AHe79DHY8GHF8DCPr6S+QU2oiv06Ch069g5euB/Jr+haC
+ENTVXGzfRUhYXW0dpVymZIXVV6Bl3Q8WnZstNoVimVw81M9elcqVgLgWbK8S1VEm
+PrIvPN/JdpCnO6EakYrdTflEOBMODtDYhMVY7WiFmzzPJRTqXrzXqxukr0ZlEnU9
+QKAatG92XqQ2AUVAkP7bFQyvfL+xfOypahKFS6TRxmFBK3Y+92Cl1FaIpGFP5Dpb
+vqgCv37TIBmJMUVpAtOnbvOYQNs6OtP0hKYuH6drJvfVR5vj8qGY65tBGzwjWEFb
+Ce1Pc4RPNbxn7T2GdVfidM0YsTK1H7uIvXrTl4EuYHTp5FocNrio0RfS9YUaExZL
+Dp7CWR0NsYV2a0D63Qo2z1vaU4tFktfEeG/DY7GOdykDWM0pRVLFAZUtRm3TUK9W
+bGsM8tW/LnOYFl6NFYrI6Fij3GBrR25yJtIHOUz4Pcc/kzXvM4Kyl8cf58JzM18L
+bFGK5wuqt1d+6l3vdj103cOqRz7pTGjnYRKaof97HW1dtBjuHfe860S/7GEdT5qK
+8cUK+1gSNfyTXZ18DWuU556sPHa8H/1kmNU2A0u4t8UC0RKDhaDkyqesuTKhsf8s
+3pptSDeciOdYdyhAsS9/joQ6mvrj+Re4ywNXAvH6v1Vd9Djec/xRmnQiOXtMg2/Q
+BKf1IEm7x70WUdyl528pI/v/Sd3I3ttBcu42/sxdW8Mjnh3i0nJjk3OD2LXslibV
+vpy/sLySOONTTGaUQtKgFPNBL4pSFGkraN03/oYT2UW1P+6K3DqNe3Ik0nZvf+uH
+oCvJVo2i1DEQONDmn9sRsD6k+Y67QbeHxM3iuQKkFUX1BsXTlSlKlHQdv55lEDwK
+v7nC2SamQMl2eoGsrInaW4MuxiJ8Y7hDbd5dGRKx6uHT9vqgTOrjKDKQn05z1NAz
+LAT6P9vCCBbXe6dOUtIMgweeXztd1yO/i4Pj73en/HwdD39xxG2zTB7lMe/dmkH5
+O1SzYnSIRbgxgozjyw5zXuIOr7AJfLI4oU4f6zurziv27k7TK3gQJq5pf4kaVIuR
+zstA3AhpcNjIee5FYVm9oYFWyX0vC7vO+QvqNG49m9TjeT+Mk4fmwxcpb/KWvTFS
+PylD/hefOA/d+BcZ8DdOLQKYNh4JwHlHw61Awd84NQgT3vD98GhbZSaRBMvg88Wt
+eFX/lojfMvF7cMEzhj6XqUEzzrDsJLEj5H4B+Y6/FsRL/vUd4Er9vUa+nd0LrMQ4
+xCbhyqxD7j6A5YfWvU2TovUK0Qfg4FygWhQ2++yyI58fNNM4wMn0cMGnSeq6qk55
+/IrkQ8MvAeegjSKZhRJwHhb4DwyPEZhbLA9kdpHUqY8EyRarIy/FV1zfpfZou5H1
+PrLt4FUPwFJ/wHsjcMI+MOX+MH1IPTfefkyjg2/3AT7oD5x5ltcNedgfspGnUS+Y
+N/1hYvuDuzR49fh/AwAA//9Qx9WryDMBAA==
+-----END COZY ASSET-----
+-----BEGIN COZY ASSET-----
+Name: /css/cozy-ui.min.css
```

Comment:
I don't think it is normal to have `/css/cozy-ui.min.css` twice!

Reply:
Nice pick, fixed !